dask_expr/reductions.py

```diff
@@ -176,7 +176,22 @@ def chunk_kwargs(self):
         return {"ignore_index": self.ignore_index, **self._subset_kwargs()}
 
     def _simplify_up(self, parent):
-        return
+        if self.subset is not None and isinstance(parent, Projection):
+            columns = parent.operand("columns")
+            if not isinstance(columns, list) and not hasattr(columns, "dtype"):
+                columns = [columns]
```

Comment:
I'll bet that this is just 

```suggestion
            columns = parent.columns
```